cmd/crio/config.go

```diff
@@ -131,6 +131,13 @@ image_volumes = "{{ .ImageVolumes }}"
 insecure_registries = [
 {{ range $opt := .InsecureRegistries }}{{ printf "\t%q,\n" $opt }}{{ end }}]
 
+# registries is used to specify registries to be used when pulling an unqualified
+# image (e.g. fedora:rawhide). "docker.io" is left here as a default registry.
+# Users can remove it and use whatever default registry they need.
+registries = [
+  "docker.io",
```

Comment:
@rhatdan could you provide an example of using this PR and having the double "docker.io" entry there?